### ISSUE TITLE
Main Blog Page

### ISSUE THREAD

**Issue by salexzee** [TribalDev/tribal_blog#12]
The main blog page is boring... as hell. What do we want to do with that? Benji and I discussed these cards that flip over. On the front having an image and the title. On the back having a short description. Maybe the first few sentences of the blog or something. What does everyone else think about that?

**Comment by salexzee** [comment]
Also, possibly infinite scroll at some stage.

**Comment by jaeming** [comment]
infinite scroll is there!

**Comment by RoadBytes** [comment]
Amaze!

**Comment by jaeming** [comment]
I think we could either do a smaller card for each post or possibly use something like masonry: http://masonry.desandro.com/
These things are heavily dependant on images to look cool. Maybe we could define some categories with a theme/image for each.

Alternatively, A more simple approach would be to do like sitepoint: http://www.sitepoint.com/
They have like a simple card with title, category, author, date, etc...

**Comment by salexzee** [comment]
I really like the sitepoint one. Works well with and without images.

**Comment by jaeming** [comment]
Agreed. I don't want to copy them too explicitly, but we already have the about cards so we could something similar (maybe smaller than out about cards).
